src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -1,217 +1,61 @@
 import ModLoaderPackageMapping from '../../../model/installing/ModLoaderPackageMapping';
-import { PackageLoader } from '../../../model/installing/PackageLoader';
+import {
+    GetInstallerVariantFromName,
+    PackageLoader,
+} from '../../../model/installing/PackageLoader';
 import VersionNumber from '../../../model/VersionNumber';
+import { EcosystemSchema } from '../../../model/schema/ThunderstoreSchema';
 
+export type GAME_NAME = string;
```

Comment:
Why is this is an exported type? I might be missing something but it doesn't look like we use it anywhere.